nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -394,17 +425,18 @@ def filter_func(point: StatisticPoint) -> bool:
                 if quantization_target_point.is_weight_target_point():
                     # If the nodes share one weight tensor, we should have only one quantizer on that
                     weights_name = self._backend_entity.get_weight_name(nncf_graph, quantization_target_point)
-
                     if weights_name in weight_layer_names:
                         continue
-                    weight_layer_names.add(weights_name)
+                    is_first_layer = len(weight_layer_names) == 0
```

Comment:
Have you tested this condition in the case of the multiple inputs and their corresponding weighted layers? It wouldn't work correctly, as I can see.

Reply:
The question is what should be the right way of applying overflow_fix in such a scenario?
1) Should only the one - first quantized weight be impacted?
2)Or should all the first quantized weights after every model input be impacted?

Reply:
After every input, I assume. What is the difference, in terms of the run-time, between the Convolution after `Input_1` and the Convolution after `Input_2`?

Reply:
Done